src/main/java/hu/dungeonhunter/Main.java

```diff
@@ -0,0 +1,40 @@
+package hu.dungeonhunter;
+
+import hu.dungeonhunter.tools.Fight;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+class Main {
+
+    public static void main(String[] args) {
+        actionMenu();
+
+    }
+
+    public static void actionMenu() {
+        Fight fight = new Fight();
+        fight.start();
+        while (fight.nextTurn()) {
+            System.out.println("\nChoose one of the following actions:");
+            System.out.println("1. Attack");
```

Comment:
You are printing the menu here and at row 36 as well. This will result lots of duplications when the menu is extended with more menu points. Let's create a writeMenu() that prints this and call this method here and in row 36.